manim/mobject/mobject.py

```diff
@@ -225,7 +225,17 @@ def construct(self):
 
     @property
     def animate(self):
-        """Used to animate the application of a method.
+        """Used to animate the application of any method of :code:`self`.
+
+        Any method called on :code:`animate` is converted to an animation of applying that method on the mobject itself.
+
+        For example, :code:`square.set_fill(WHITE)` sets the fill color of a square, while :code:`sqaure.animate.set_fill(WHITE)` animates this action.
```

Comment:
These two lines are a bit long, please break them:

```suggestion
        Any method called on :code:`animate` is converted to an animation of applying
        that method on the mobject itself.

        For example, :code:`square.set_fill(WHITE)` sets the fill color of a square,
        while :code:`sqaure.animate.set_fill(WHITE)` animates this action.
```